modules/nw-infw-operator-installing.adoc

```diff
@@ -0,0 +1,151 @@
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+. To verify that the Operator is installed, enter the following command:
++
+[source,terminal]
+----
+$ oc get ip -n openshift-ingress-node-firewall
+----
++
+.Example output
+[source,terminal]
+----
+NAME            CSV                                         APPROVAL    APPROVED
+install-5cvnz   ingress-node-firewall.4.12.0-202211122336   Automatic   true
+----
+
+. To verify the version of the Operator, enter the following command:
+
++
+[source,terminal]
+----
+$ oc get csv -n openshift-ingress-node-firewall
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                        DISPLAY                          VERSION               REPLACES                                    PHASE
+ingress-node-firewall.4.12.0-202211122336   Ingress Node Firewall Operator   4.12.0-202211122336   ingress-node-firewall.4.12.0-202211102047   Succeeded
+----
+
+[id="install-operator-web-console_{context}"]
+== Installing the Ingress Node Firewall Operator using the web console
+
+As a cluster administrator, you can install the Operator using the web console.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have an account with administrator privileges.
+
+.Procedure
+
+
+. Install the Ingress Node Firewall Operator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Select *Ingress Node Firewall Operator* from the list of available Operators, and then click *Install*.
+
+.. On the *Install Operator* page, under *Installed Namespace*, select *Operator recommended Namespace*.
+
+.. Click *Install*.
+
+. Verify that the Ingress Node Firewall Operator is installed successfully:
+
+.. Navigate to the *Operators* -> *Installed Operators* page.
+
+.. Ensure that *Ingress Node Firewall Operator* is listed in the *openshift-ingress-node-firewall* project with a *Status* of *InstallSucceeded*.
++
+[NOTE]
+====
+During installation an Operator might display a *Failed* status.
+If the installation later succeeds with an *InstallSucceeded* message, you can ignore the *Failed* message.
+====
+
++
+If the Operator does not have a *Status* of *InstallSucceeded*, troubleshoot using the following steps:
+
++
+* Inspect the *Operator Subscriptions* and *Install Plans* tabs for any failures or errors under *Status*.
+* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the `openshift-ingress-node-firewall` project.
+* Check the namespace of the YAML file. If the annotation is missing, you can add the annotation `workload.openshift.io/allowed=management` to the Operator namespace with the following command:
++
+[source,terminal]
+----
+$ oc annotate ns/openshift-ingress-node-firewall-operator workload.openshift.io/allowed=management
+----
++
+[NOTE]
+====
+For {sno} clusters, the `openshift-ingress-node-firewall-operator` namespace requires the `workload.openshift.io/allowed=management` annotation.
```

Comment:
openshift-ingress-node-firewall instead of  openshift-ingress-node-firewall-operator.